tests/functional/simple_snapshot/test_hard_delete_snapshot.py

```diff
@@ -94,7 +82,13 @@ def test_snapshot_hard_delete(project):
     for result in snapshotted[10:]:
         # result is a tuple, the dbt_valid_to column is the latest
         assert isinstance(result[-1], datetime)
-        assert result[-1].replace(tzinfo=currenttz()) >= invalidated_snapshot_datetime
+        # Plenty of wiggle room if clocks aren't perfectly sync'd, etc
+        tolerance = timedelta(minutes=1)
+        assert (
+            result[-1].replace(tzinfo=pytz.UTC) > (invalidated_snapshot_datetime - tolerance)
+        ) and (
+            result[-1].replace(tzinfo=pytz.UTC) < (invalidated_snapshot_datetime + tolerance)
+        ), f"SQL timestamp {result[-1].replace(tzinfo=pytz.UTC).isoformat()} is not close enough to Python UTC {result[-1].replace(tzinfo=pytz.UTC).isoformat()}"
```

Comment:
```suggestion
        ), f"SQL timestamp {result[-1].replace(tzinfo=pytz.UTC).isoformat()} is not close enough to Python UTC {invalidated_snapshot_datetime.isoformat()}"
```

Reply:
Resolved in [c736197](https://github.com/dbt-labs/dbt-core/pull/7665/commits/c7361976f9b0dcab63059000759a89728d64d9ab)